Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkDemos/SR-ResearchDemos/StereoPicture/EyeLink_StereoPicture.m

```diff
@@ -0,0 +1,408 @@
+        % xs, ys: top-left corner of the region to be transferred within the source image
+        % width, height: size of region to be transferred within the source image (note, values of 0 will include the entire width/height)
+        % xd, yd: location (top-left) where image region to be transferred will be presented on the Host PC
+        % This image transfer function works for non-resized image presentation only. If you need to resize images and use this function please resize
+        % the original image files beforehand
+        imgData = imread(imgNameLeft); % Get image file data
+        transferStatus = Eyelink('ImageTransfer', imgData, 0, 0, 0, 0, round(width/2-imgInfo.Width/2), round(height/2-imgInfo.Height/2));    
+        if dummymode == 0 && transferStatus ~= 0 % If connected to EyeLink and image transfer fails
+            fprintf('Image transfer Failed\n'); % Print some text in Matlab's Command Window
+        end
+
+        % Optional: draw feedback box on Host PC interface instead of (or on top of) backdrop image.
+        % See section 25.7 'Drawing Commands' in the EyeLink Programmers Guide manual
+        Eyelink('Command', 'draw_box %d %d %d %d 15', round(width/2-imgInfo.Width/2), round(height/2-imgInfo.Height/2), round(width/2+imgInfo.Width/2), round(height/2+imgInfo.Height/2));
+
+        % Perform a drift check/correction. If using an EyeLink I or II a drift correction is performed by default
+        % Optionally provide x y target location, otherwise target is presented on screen centre
+        EyelinkDoDriftCorrection(el, round(width/2), round(height/2));
+                
+        %STEP 5.2: START RECORDING
+        
+        % Put tracker in idle/offline mode before recording. Eyelink('SetOfflineMode') is recommended 
+        % however if Eyelink('Command', 'set_idle_mode') is used allow 50ms before recording as shown in the commented code:        
+        % Eyelink('Command', 'set_idle_mode');% Put tracker in idle/offline mode before recording
+        % WaitSecs(0.05); % Allow some time for transition           
+        Eyelink('SetOfflineMode');% Put tracker in idle/offline mode before recording
+        Eyelink('StartRecording'); % Start tracker recording
+        WaitSecs(0.1); % Allow some time to record a few samples before presenting first stimulus
+        
+        % STEP 5.3: PRESENT STIMULUS; CREATE DATAVIEWER BACKDROP AND INTEREST AREA
+        
+        % Present initial trial image        
+        imgTexture = zeros(1,2); % Preallocate variable
+        Screen('FillRect', window, el.backgroundcolour);% Prepare grey background on backbuffer
+        for itScr = 0:1 % iterate through left/right eye windows
+            Screen('SelectStereoDrawBuffer', window, itScr); % select left or right eye window           
+            if itScr == 0
+                % imgData = imread(imgData); % Read left eye image done above, don't need to repeat

```

Comment:
Bit clunky to tell them what they should not do, then not do it. Maybe better only have the else branch, and add a comment there why it is done the way it is done?

Reply:
understood and agreed the StereoMode demo needs to be revamped to reflect the most current PTB conventions. I will include this as a priority item in the next round of revisions following completion of the rest of this pull request to allow for additional time to make those changes and thorough testing before publishing